docs/guide/SecurityBC.md

```diff
@@ -273,3 +273,234 @@ Here is the response that comes back:
+              - text/html
+secret:
+  # without this we would need to put the JWKS in the config, which would mean we couldn't rotate it just by changing the secret
+  manage: false
+  # change to whatever is used for secret name
+  name: oathkeeper-jwks
+deployment:
+  extraEnv:
+    # for whatever reason this environment variable only gets set if the JWKS is in the config even though the rest of the secret mounting
+    # and such still happens
+    - name: MUTATORS_ID_TOKEN_CONFIG_JWKS_URL
+    value: file:///etc/secrets/mutator.id_token.jwks.json
+
+
+```
+
+### Rules
+
+Rule resources will need to be created in k8s along the lines of the following, for each backend “match” (URL regex plus HTTP method(s)) protected with a permission.
+
+As the flexibility to define 3rd party services and bounded contexts increases, they can define their own rules (perhaps behind a helm values flag), which express which permissions should be required for which URLs. 
+
+The biggest potential issue here is that each match MUST be unique. If a request matches multiple, Oathkeeper will complain. Once a general pattern is picked that results in unique regexes, this will not happen except with user error.
+
+```yaml
+
+apiVersion: oathkeeper.ory.sh/v1alpha1
+kind: Rule
+metadata:
+  name: a-unique-name
+spec:
+  version: v0.36.0-beta.4
+  upstream:
+    # set to whatever URL this request should be forwarded to
+    url: http://internal-url-of-backend-service/
+  match:
+    # this might need to be http even if external is https, it depends on how ingress does things
+    # my recommendation is to have a given prefix, then the "everything else in the domain name" matcher
+    # so it doesn't need to be changed when the config is moved between various main domains
+    # then whatever is needed for the specific path (this is set to match all subpaths)
+    # regexes go in between angle brackets
+    url: https://example.<[^/]*>/<.*>
+    methods:
+    # whatever method(s) this rule applies to
+    - GET
+  authenticators:
+    - handler: oauth2_introspection
+    # comment out this second one to not allow browser-cookie access
+    - handler: cookie_session
+  authorizer:
+    handler: remote_json
+    # these will generally be identical for all rules,
+    # except "object" will be changed to the permission ID that is relevant for
+    # this URL
+    payload: |
+    {
+      "namespace": "permission",
+      "object": "PERMISSION IDENTIFIER HERE",
+      "relation": "granted",
+      "subject_id": "{{ print .Subject }}"
+    }
+  mutators:
+    # change this to an empty array if the id_token isn't needed, if you want
+    - handler: id_token
+
+```
+### Configure Oathkeeper to use Kratos as it's cookie Authenticator
+[docs here](https://www.ory.sh/oathkeeper/docs/next/pipeline/authn)
+
+### Configure Oathkeeper to use WSO2 ISKM for token introspection
+Check that a token is valid
+[docs here](https://www.ory.sh/oathkeeper/docs/next/pipeline/authn)

```

Comment:
Please can you make this reference a little bit more informative? What details will users find in the referenced doc that helps them with token introspection?

Reply:
I have added to these descriptions